composer/loggers/progress_bar_logger.py

```diff
@@ -19,7 +19,14 @@
 
 __all__ = ['ProgressBarLogger']
 
-_IS_TRAIN_TO_KEYS_TO_LOG = {True: ['loss/train'], False: ['metrics/eval/Accuracy']}
+
+class FormatDict(dict):
```

Comment:
Hmm, I would prefer not to subclass dict. Could we replace this with a function that takes a dict and returns a dict?